src/docker_build.sh

```diff
@@ -5,19 +5,23 @@ set -e
 VERSION=$(cat VERSION)
 
 ## Choose the docker base image to build from:
+## 0. (For developer) If specified on the command line, use that.
 ## 1. (For developer) If broadinstitute/slurm_gcp_docker_base:$VERSION exists locally, use it.
 ##    This is the local image built by docker_build_base.sh
 ## 2. Otherwise use gcr.io/broad-getzlab-workflows/slurm_gcp_docker_base:$VERSION
 
-#if sudo docker image inspect broadinstitute/slurm_gcp_docker_base:$VERSION >/dev/null 2>/dev/null; then
-#    echo "Building docker image from broadinstitute/slurm_gcp_docker_base:$VERSION (local)"
-#    docker_base_image=broadinstitute/slurm_gcp_docker_base:$VERSION
-#else
-#    echo "Building docker image from gcr.io/broad-getzlab-workflows/slurm_gcp_docker_base:$VERSION"
-#    docker_base_image=gcr.io/broad-getzlab-workflows/slurm_gcp_docker_base:$VERSION
-#fi
-
-docker_base_image=$(cat DOCKER_SRC)
+if [ $# -gt 0 ]; then
+	# TODO: allow version to be specified too
+	docker_base_image=$1
```

Comment:
I like it. Though we should pass this argument (probably via environment variable?) to `./generate_container_host_image.py` as well. https://github.com/getzlab/slurm_gcp_docker/pull/16/files#diff-e50e28e0880f300b78a4db029b88332358ff6f54021b9269383b607da1fdc816R117-R121

Reply:
Oh good catch!